src/client/hud.cpp

```diff
@@ -353,12 +355,21 @@ void Hud::drawLuaElements(const v3s16 &camera_offset)
 	std::vector<HudElement*> elems;
 	elems.reserve(player->maxHudId());
 
-	// Add builtin minimap if the server doesn't send it.
+	// Add builtin elements if the server doesn't send them.
+	// Declared here such that they have the same lifetime as the elems vector
 	HudElement minimap;
-	if (client->getProtoVersion() < 44 && (player->hud_flags & HUD_FLAG_MINIMAP_VISIBLE)) {
-		minimap = {HUD_ELEM_MINIMAP, v2f(1, 0), "", v2f(), "", 0 , 0, 0, v2f(-1, 1),
-				v2f(-10, 10), v3f(), v2s32(256, 256), 0, "", 0};
-		elems.push_back(&minimap);
+	HudElement hotbar;
+	if (client->getProtoVersion() < 44) {
```

Comment:
I rebased, and everything should still work fine, but before it can be merged we need a protocol bump, since version 5.10.0 already got released.

Reply:
The protocol bump is usually done by the PR that needs the bump, see https://github.com/minetest/minetest/commit/0d61598d8aef6466fe232e97bd0de1ed4e726a54#diff-79d750b4822ef0829179bb47723bfd05c0346ab2b8570306c28ffeb212d7efd4 for example.

Reply:
Ok, I adjusted to protocol version.